packages/cdk8s-plus/src/volume.ts

```diff
@@ -98,6 +98,20 @@ export class Volume {
     });
   }
 
+  /**
+   * Create a volume the wraps a k8s.Volume for usage in cdk8s-plus.
+   * 
+   * @param name Name of the volume.
+   * @param volume The k8s.Volume resource to wrap.
+   * 
+   * note: 
+   *    Using Omit outputs: 
+   *      jsii/compiler - Type model errors prevented the JSII assembly from being created
+   */
+  public static fromVolumeSpec(name: string, volume: any /*Omit<k8s.Volume, 'name'>*/): Volume {
```

Comment:
I would love to find a way to use Omit here but it causes an error with the jsii/compiler:

```
jsii/compiler - Type model errors prevented the JSII assembly from being created
```

Any ideas?